src/fontstash.h

```diff
@@ -920,15 +920,14 @@ int fonsAddFont(FONScontext* stash, const char* name, const char* path)
 {
 	FILE* fp = 0;
 	int dataSize = 0, readed;
-	unsigned char* data = NULL;
-
 	// Read in the font data.
 	fp = fons__fopen(path, "rb");
-	if (fp == NULL) goto error;
+	if (fp == NULL) return FONS_INVALID;
 	fseek(fp,0,SEEK_END);
 	dataSize = (int)ftell(fp);
 	fseek(fp,0,SEEK_SET);
-	data = (unsigned char*)malloc(dataSize);
+
+	unsigned char data[dataSize];
```

Comment:
> If I remember correctly, @memononen is keeping the codebase C89, so we can't use VLAs.

I believe stb_truetype doesn't support c89 (at least not without changes). So that wouldn't make much sense. I'll remove the VLA's though.

Reply:
fontstash also uses "//" for comments, so I don't think it's trying to support c89 anyway.